src/main/scripts/addon

```diff
@@ -101,6 +101,7 @@ fi
 
 JAVA_OPTS="-Dplf.home=\"$PLF_HOME\" $EXO_AM_OPTS"
 JAVA_OPTS="$JAVA_OPTS -Duser.language=\"en\""
+JAVA_OPTS="$JAVA_OPTS -Dam.properties.path=\"$PLF_HOME/addons/configuration/am.properties\""
```

Comment:
Why not keeping the default settings in the jar ?

Reply:
Because we need to define a remote catalog per distribution.

Reply:
Yes but I was talking about the default one, for meeds.

Reply:
So that it modification can be more simple even after releasing the distribution, if ever someone wants to change the default remote addons catalog. ( Better for community troubleshooting )

Reply:
Not sure it is worth adding another file, but nevermind, not a big deal, I have my answer, thanks.